app/src/main/java/com/babylon/wallet/android/domain/usecases/signing/SignTransactionUseCase.kt

```diff
@@ -33,7 +33,7 @@ class SignTransactionUseCase @Inject constructor(
 
     suspend operator fun invoke(request: Request): Result<NotarizationResult> {
         val manifestWithLockFee = request.manifestWithLockFee
-        val summary = request.manifestWithLockFee.summary
+        val summary = request.manifestWithLockFee.summary!!
```

Comment:
`summary` is now throwing, but at the stage of signing it should be valid, a transaction with an invalid summary will be reject at preview level.

Reply:
I guess you mean that summary is nullable but at this point in time it should not. IMHO better use 
```
val summary = requireNotNull(request.manifestWithLockFee.summary)
```

Reply:
I made this change as part of my other PR on top of this one.